pkg/minikube/node/start.go

```diff
@@ -286,11 +289,16 @@ func joinCluster(starter Starter, cpBs bootstrapper.Bootstrapper, bs bootstrappe
 func Provision(cc *config.ClusterConfig, n *config.Node, apiServer bool, delOnFail bool) (command.Runner, bool, libmachine.API, *host.Host, error) {
 	register.Reg.SetStep(register.StartingNode)
 	name := config.MachineName(*cc, *n)
-	if apiServer {
-		out.Step(style.ThumbsUp, "Starting control plane node {{.name}} in cluster {{.cluster}}", out.V{"name": name, "cluster": cc.Name})
-	} else {
-		out.Step(style.ThumbsUp, "Starting node {{.name}} in cluster {{.cluster}}", out.V{"name": name, "cluster": cc.Name})
+
+	startingPhrase := "Starting control plane node"
+	if !apiServer {
+		startingPhrase = "Starting worker node"
 	}
+	if cc.KubernetesConfig.KubernetesVersion == constants.NoKubernetesVersion {
+		startingPhrase = "Starting minikube without Kubernetes"
+	}
+
+	out.Step(style.ThumbsUp, "{{.nodeName}} {{.name}} in cluster {{.cluster}}", out.V{"nodeName": startingPhrase, "name": name, "cluster": cc.Name})
```

Comment:
constructing the sentence this way makes it very difficult for internationalization, since templated terms like {{.nodeName}} get ignored by design

Reply:
I see you are right I think we better of with more verbose code for sake of translation, will fix it